app/settings.py

```diff
@@ -44,7 +44,8 @@ def get_key(_key_name):
 EQ_GIT_REF = os.getenv('EQ_GIT_REF', 'unknown')
 EQ_NEW_RELIC_CONFIG_FILE = os.getenv('EQ_NEW_RELIC_CONFIG_FILE', './newrelic.ini')
 EQ_SCHEMA_DIRECTORY = os.getenv('EQ_SCHEMA_DIRECTORY', 'app/data')
-EQ_SESSION_TIMEOUT = int(os.getenv('EQ_SESSION_TIMEOUT', '28800'))
+EQ_SESSION_TIMEOUT = int(os.getenv('EQ_SESSION_TIMEOUT', '2700'))
```

Comment:
I recommend adding _SECONDS to the name so it's clear this is in seconds, when dealing with times like this it's always hard to know if its milliseconds or not. Same below.